mitosheet/mitosheet/public/v3/formatting.py

```diff
@@ -70,4 +72,18 @@ def add_formatting_to_excel_sheet(
                     if row % 2 == 0:
                         sheet.cell(row=row, column=col).style = even_name
                     else:
-                        sheet.cell(row=row, column=col).style = odd_name
\ No newline at end of file
+                        sheet.cell(row=row, column=col).style = odd_name
+
+        # Add conditional formatting
+        if conditional_formats is None:
```

Comment:
Maybe let's break this out into a little helper?

Reply:
Good idea